roboteam_ai/src/stp/plays/offensive/Attack.cpp

```diff
@@ -100,20 +100,21 @@ void Attack::calculateInfoForRoles() noexcept {
         enemyMap.insert({score, enemy->getPos()});
     }
 
-    constexpr auto midfielderNames = std::array{"pass_defender_1", "pass_defender_2", "pass_defender_3"};
-    auto activeMidfielderNames = std::vector<std::string>{};
-    for (auto name : midfielderNames) {
-        if (stpInfos[name].getRobot().has_value()) activeMidfielderNames.emplace_back(name);
+    if (enemyMap.size() < 3) {
```

Comment:
What if there are two opponents in the field? Are these left undefended?

Reply:
This is a play in which we have ball possession, so if the opponents only have 2 robots it should be fine to leave them undefended as they will not be able to shoot unless they take the ball.